This is a testharness.js-based test.
FAIL history pushState SECURITY_ERR assert_throws: function "function () {
        window.history.pushState(1, document.title, 'http://www.microsoft.com/test.html');
      }" did not throw
Harness: the test ran to completion.

